test/relay_hub_test.js

```diff
@@ -760,6 +761,48 @@ contract("RelayHub", function (accounts) {
 
     });
 
+    it("should revert the 'relayedCall' if 'preRelayedCall' reverts", async function () {
```

Comment:
TODO: duplicate the test to check for postRelayed reverted

Reply:
I think this test is actually a copy of a test that checks 'postRelayed', I added this test last week.